turbo.json

```diff
@@ -10,20 +10,12 @@
     "clean": {
       "cache": true
     },
-    "storybook-build": {
-      "outputs": [
-        "apps/pie-storybook/build/**"
-      ],
-      "dependsOn": [
-        "^build"
-      ]
-    },
-    "storybook-serve": {
+    "pie-storybook#serve": {
```

Comment:
More for my own knowledge – is this how you target specific serve's in each package in turbo?

Reply:
Yep. For example, if we ever add any more `serve` commands, it allows you to target specific packages `serve` command to rely on specific package `build` tasks :)